server.js

```diff
@@ -24,6 +26,8 @@ mongoose
     .connect(db, { useNewUrlParser: true })
     .then(() => {
         console.log('MongoDB Connected');
+        // User.findOne({ email: 'danibtechnology@gmail.com'})
+        // 	.then(user => console.log(user));
```

Comment:
Not sure if comment here is necessary.

Reply:
I used that for debugging. It's not necessary, indeed.